src/Model/GroundWaterFlow/gwf3sfr8.f90

```diff
@@ -2499,23 +2503,27 @@ subroutine sfr_bd_obs(this)
           call this%obs%SaveOneSimval(obsrv, v)
         end do
       end do
-    end if
-    !
-    ! -- write summary of package block error messages
-    if (count_errors() > 0) then
-      call this%parser%StoreErrorUnit()
-      call ustop()
+      !
```

Comment:
Are there line ending issues here?  Why is this code showing up as changed?

Reply:
There was only a single comment line before. Added

```
!
! -- write summary of package error messages
```